ironfish/src/metrics/meter.ts

```diff
@@ -37,11 +41,18 @@ export class Meter {
 
   constructor() {
     this._intervalMs = 1000
+
     this._rate1s = new EwmAverage(this.METER_TIME_INTERVALS_MS.rate1s / this._intervalMs)
     this._rate5s = new EwmAverage(this.METER_TIME_INTERVALS_MS.rate5s / this._intervalMs)
     this._rate1m = new EwmAverage(this.METER_TIME_INTERVALS_MS.rate1m / this._intervalMs)
     this._rate5m = new EwmAverage(this.METER_TIME_INTERVALS_MS.rate5m / this._intervalMs)
+
     this._average = new RollingAverage(1000)
+
+    this._rollingRate1s = new RollingAverage(1000 / this._intervalMs)
+    this._rollingRate5s = new RollingAverage(5000 / this._intervalMs)
+    this._rollingRate1m = new RollingAverage((1 * 60 * 1000) / this._intervalMs)
+    this._rollingRate5m = new RollingAverage((5 * 60 * 1000) / this._intervalMs)
```

Comment:
Want to use `this.METER_TIME_INTERVALS_MS` for these? It's used right above in the EwmAverage and seems to be the same

Reply:
Yea I'll use it. Although not really sure why it was needed in the first place. Just seems like more code indirection. Now that we have two places using it though I guess it helps 🤷🏻

Reply:
Talked about this in a call, I'm okay with getting rid of the variables.